circle.yml

```diff
@@ -18,3 +18,6 @@ test:
     - nvm use 4; npm rebuild &> /dev/null; npm run testci
     - nvm use 6; npm rebuild &> /dev/null; npm run testci
     - nvm use 8; npm rebuild &> /dev/null; npm run testci
+  post:
+    - npm test
+    - ./node_modules/.bin/nyc report --reporter=lcov > coverage.lcov && bash <(curl -s https://codecov.io/bash)
```

Comment:
Should this just be `npm run report-coverage` (from https://github.com/segmentio/analytics-node/pull/148/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R30)?